_pkgdown.yml

```
@@ -96,95 +96,151 @@ news:
     href: https://shiny.posit.co/blog/posts/announcing-new-r-shiny-ui-components/
 
 reference:
-- title: Create a dashboard
-  contents:
+- title: Page Layouts
+  desc: >
+    Page layouts are the basic building blocks of any UI. bslib pages always
+    include Bootstrap and are can be themed with bslib's theming capabilities.
+- subtitle: Dashboard layouts
+  desc: >
+    These page layouts work best for dashboards. For a single-page dashboards
+    with an optional sidebar, try `page_sidebar()`. For multi-tab dashboards
+    organized as pages in a navbar, also with an optional sidebar, try
+    `page_navbar()`. For a full-page dashboard with items that fill the screen,
+    try `page_fillable()`.
+- contents:
     - page_sidebar
-- title: Page layouts
-  contents:
+    - page_navbar
+    - page_fillable
+- subtitle: Basic page layouts
+  desc: >
+    bslib variants of classic Shiny page layouts.
+- contents:
     - page
-- title: Theme creation
-  description: |
-    Tools for creating customized Bootstrap themes.
+
+- title: User Interface Layouts
+  desc: >
+    These functions help you layout the user interface (UI) elements of your
+    app.
+- subtitle: Layout multiple UI elements
+  desc: |
+    Organize inputs and outputs into Bootstrap's
+    [12-column CSS grid](https://getbootstrap.com/docs/5.3/layout/css-grid/)
+    with `layout_columns()` or into a responsive, column-first grid layout with
+    `layout_column_wrap()`. Place a sidebar next to any UI elements with
+    `layout_sidebar()`.
   contents:
-    - bs_theme
-    - bs_add_variables
-    - font_face
-- title: Interactive theming
-  description: |
-    Interactively customize themes.
+    - layout_columns
+    - layout_column_wrap
+    - layout_sidebar
+    - breakpoints
+- subtitle: Navigation
```

Would have the benefit of matching up with the "Layouts" dropdown in the navbar

```suggestion
- subtitle: Multiple panels
```